config/local_node.json

```diff
@@ -3,7 +3,7 @@
         "0.0.3": "127.0.0.1:50211"
     },
     "mirrorNetwork": [
-        "127.0.0.1:5600"
+        "127.0.0.1:5551"
```

Comment:
This will break our other queries that talk directly to the gRPC mirror node endpoint. Since you've only got one test for the `HttpClient` you'd probably be fine to just hardcode this address in the test for now, but if we add more tests down the line we could try to come up with a better way to distinguish between different mirror node endpoints in the config file here. We definitely don't want to change this address here though.

Reply:
with the current local node:
Consensus Node Url - 127.0.0.1:50211
Node Id - 0.0.3
Mirror Node Url - http://127.0.0.1:5551

Reply:
Talked offline, but there are several different endpoints for mirror node, `5551` is the REST API while `5600` is the gRPC endpoint. Leaving this as is and will hardcode the REST API URL for now. Will create a new issue down the line to update the config file to have both URLs if need be